source/utilities.tex

```diff
@@ -5186,18 +5192,26 @@
 \throws Any exception thrown by the selected constructor of \tcode{T}.
 
 \pnum
-\remarks This constructor shall not participate in overload resolution
-unless tcode{is_constructible_v<T, Args...>} is \tcode{true}.
+\remarks
+This constructor shall not participate in overload resolution unless
+\tcode{is_copy_constructible_v<T>} is \tcode{true}
+and \tcode{is_constructible_v<T, Args...>} is \tcode{true}.
```

Comment:
fyi, folks usually put the conjunction ("and" in this case) at the end of the previous line.  not that you need to change it here, but I try to keep the spec coding style consistent (I'm sure you've noticed we have a long way to go there!)